.github/workflows/run-translation-tests.yml

```diff
@@ -3,7 +3,9 @@ name: Run notebook translation tests
 on:
   push:
     paths:
-    - 'examples/tutorials/**'
+    - "examples/tutorials/**"
+  pull_request:
```

Comment:
I think this `pull_request` section also needs to specify the paths.

Reply:
> Thanks for taking this on! Looks like it’s running the translation tests on this PR, which it shouldn’t. I think it just needs to specify the paths for both pushes and pull requests.

Yes, I removed the changes I made to the notebook and it still triggers the checks. Will try and change that.

Reply:
I will revert the last commit if everything is fine.

> Looks like it’s running the translation tests on this PR, which it shouldn’t.
`examples/tutorials/**` includes all files in tutorials. If you want only the notebook files then it should be `example/tutorials/*.ipynb`

Reply:
I think we do want everything in that directory (recursively) to trigger the translation tests. Does `**` cover that? Do we need `**/*`?

Reply:
no, `**` should work and include everything in `tutorials/`